docs/Playing/LT-Guide/twenty-turns.rst

```diff
@@ -0,0 +1,604 @@
+.. SPDX-License-Identifier: GPL-3.0-or-later
+.. SPDX-FileCopyrightText: James Robertson <jwrober@gmail.com>
+
+.. Custom Interpretive Text Roles for longturn.net/Freeciv21
+.. role:: unit
+.. role:: improvement
+.. role:: wonder
+.. role:: advance
+
+The Next 20 Turns
+*****************
+
+The early game is one of expansion. In most Longturn Traditional (LTT) or Longturn Experimental (LTX)
+multiplayer rules-based games you do not need to worry so much about putting a defender(s) into your new
+cities right away. Start with :unit:`Workers` to develop the land and build roads between cities. The quicker
+you can upgrade the :unit:`Tribal Workers` to regular :unit:`Workers` is also important. The
+:unit:`Tribal Workers` do not gain veteran status and actually have less “work points” than regular
+:unit:`Workers`. Refer to `Managing your Workforce`_ below for more information.
+
+.. note::
+  Many early forms of government support martial law to help with happiness. This is definitely a place where
+  some kind of military unit is very important. See `Early Government`_ section below.
+
+
+Exploring the Area: Opening up the Map
+======================================
+
+Every player starts with two (2) :unit:`Explorers`. With the discovery of :advance:`Seafaring` a player can
+build more if needed (or replace one that was killed by an enemy). :unit:`Explorers` are often your first form
+of in game diplomacy. As you move your :unit:`Explorers` around (ideally from mountain to mountain for the
+extra vision) you are going to find your neighbors. :term:`LTT`/:term:`LTX` rules enable a “contact” embassy
+to be created for the current turn if your :unit:`Explorers` comes into contact with another nation’s unit (of
+any kind). For an :unit:`Explorer` to come into “contact” with another unit, they simply are adjacent to each
+other.
+
+This is a good opportunity to do two things:
+
+#. Reach out to the player in game chat or on Discord and start a dialog. Ask permission to explore their
+   territory. This is the best way to A) not aggravate your neighbor and B) keep your :unit:`Explorers` alive.
+
+#. In the game, open the diplomacy screen via the “meet” button on the
+   :ref:`Nations view <game-manual-nations-and-diplomacy-view>` and ask to swap embassies. Many players will
+   agree to the sharing of embassies as it gives important information that both sides can equally use. This
+   is why it is important to do #1 first, so you have already started a dialog with your neighbor.
+
+At a minimum, find out where your neighbor’s capital city and major borders are. It goes without saying that
+you should not overstay your welcome. There is also nothing wrong with placing an :unit:`Explorers` on a
+shared border, on a tile inside your border, to act as a sentry.
+
+.. note::
+  As opposed to typical single-player :term:`AI` games, “peace” is the default state between nations in
```

Comment:
Does it still hold true, that peace is the default state? I got the impression, that we are moving towards "war" as the default, because Armistice is a "newbie trap".

Reply:
Good catch. In LTT its Armistice->Peace and in LTX its War. I'll update the other branch with a change.